### DISCUSSION TITLE
Screened Poisson generates empty mesh

### DISCUSSION THREAD

**Discussion by Kampii** [cnr-isti-vclab/meshlab#1225] [Q&A] [answered]
Hello everyone! I am pulling my hair out and I hope that someone can point me what I am doing wrong. Please kindly see [this 1 minute video](https://youtu.be/TNSz-xilGZc) where I show the steps that I take. The point cloud that I am using can be downloaded in [this link](https://www.dropbox.com/sh/24vz2etmia5crxl/AABTGJCYk9-XNpd5xo4L-KSMa?dl=0).

**Comment by alemuntoni** [comment, marked as answer]
It is a bug.
Opened issue #1227. You'll be notified on that issue when the bug will be solved.
Thanks for reporting!

**Reply by alemuntoni** [reply]
Fixed!

**Comment by Kampii** [comment]
Wow Alessandro, that was quick. I checked the https://github.com/cnr-isti-vclab/meshlab/releases but the newest version is from 21 days ago. Where do I get the updated version?

**Reply by alemuntoni** [reply]
You have to download the [last nightly](https://github.com/cnr-isti-vclab/meshlab/wiki/How-to-install-the-last-nightly-version). Be sure to download the one generated from the master branch.
